### PR TITLE
give the app more structure and add search field

### PR DESCRIPTION
**Pull request by xeronimus**

### CODE REVIEW

**Comment by xeronimus** [comment]
updated my master branch with your changes.  now again ready to merge.

File: songs/songs.js [hunk 1]
```diff
@@ -377,7 +378,7 @@ var songs = [
         artist: 'Ralph Mc Tell'
     },
     {
-        title: '99 Luftbalons',
+        title: '99 Luftballons',
```
Comment (adiherzog) [inline songs/songs.js:377]: 👍 😀

File: songs/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "name": "adiherzog.github.io",
+    "version": "0.0.2",
+    "description": "",
+    "main": "index.html",
+    "scripts": {
+        "test": "eslint './**/*.js'"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/xeronimus/adiherzog.github.io.git"
```
Comment (adiherzog) [inline songs/package.json:11]: I think we should use the original repository URL here (github.com/adiherzog).

**Comment by adiherzog** [comment]
Please imporve spacing above search field:
image

File: songs/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
```
Comment (adiherzog) [inline songs/package.json:15]: It's ok for me to use ISC as a license. Please also add a LICENCE file into the "songs" folder.

File: songs/package.json (hunk 2 above)
Comment (adiherzog) [reply songs/package.json:11]: Same applies for the other URLs in this file

Comment (xeronimus) [reply songs/package.json:11]: this is a file generated by npm itself.  the url format is therefore suggested by npm.. should we change it anyway`?

File: songs/package.json (hunk 3 above)
Comment (xeronimus) [reply songs/package.json:15]: done

**Comment by xeronimus** [comment]
everything fixed, except the urls in package.json

File: songs/package.json (hunk 2 above)
Comment (adiherzog) [reply songs/package.json:11]: Yes I would change it anyway. Or will they be overwritten by npm later, e.g. when adding a new dependency? I don't think so, right?

**Pull request merged**